ops/ami/soapbox-app.json

```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+        "aws_region": "us-east-1",
+        "instance_type": "t2.micro",
+        "ssh_username": "ec2-user",
+        "ami_name": "soapbox-aws-linux-ami"
+    },
+    "builders": [{
+        "type": "amazon-ebs",
+        "region": "{{user `aws_region`}}",
+        "source_ami_filter": {
+          "filters": {
+            "name": "amzn-ami-hvm-*",
+            "root-device-type": "ebs",
+            "virtualization-type": "hvm"
+          },
+          "owners": ["amazon"],
+          "most_recent": true
+        },
+        "instance_type": "{{user `instance_type`}}",
+        "ssh_username": "{{user `ssh_username`}}",
+        "ami_name": "{{user `ami_name`}}-{{isotime \"2016-01-02 03-04-05\"}}",
```

Comment:
I feel like less aggravation is to be had in downstream scripts and programs if we avoid spaces in names like such. Maybe replace space with a `T`?